ansible_inventory_server/utils.py

```diff
@@ -46,3 +46,30 @@ def json_response(self, data):
             response = json.dumps(data)
 
         self.write(response)
+
+
+def _ip_bits(ip):
+    """'127.0.0.1'  -->  '01111111000000000000000000000001'"""
+    ip = ip.split('/')[0]
+    return ''.join(bin(int(byte))[2:].rjust(8, '0') for byte in ip.split('.'))
+
+
+def ip_in_subnet(ip, subnet):
```

Comment:
I would prefer to issue the builtin ipaddress module. By making the assumption that the ip equals to a /32 subnet one can easily check if that subnet overlaps the given subnet from the user:
` ipaddress.ip_network(ip).overlaps(ipaddress.ip_network(subnet))`